public/js/updateBook.ts

```diff
@@ -9,52 +9,482 @@ interface Book {
     genres: string[];
 }
 
-window.onload = fetchAndDisplayBookDetails;
+const routeUrl = new URL(window.location.href);
+const isbn = routeUrl.searchParams.get('isbn');
 
-async function fetchAndDisplayBookDetails() {
-    const routeUrl = new URL(window.location.href);
+document.addEventListener("DOMContentLoaded", function () {
+    fetchAndDisplayBookDetails();
 
-    const isbn = routeUrl.searchParams.get('isbn');
+    async function fetchAndDisplayBookDetails() {
+        const url = `/books/${isbn}`;
```

Comment:
Is there no way to use the 'search' endpoint with the isbn number?
Since we ensure the user cannot add another book with the same isbn number, then the 'search using isbn' should only give one book...